plugins/saveactions.c

```diff
@@ -568,41 +1173,51 @@ static void configure_response_cb(GtkDialog *dialog, gint response, G_GNUC_UNUSE
 			}
 		}
 
-		g_key_file_set_integer(config, "backupcopy", "dir_levels", backupcopy_dir_levels);
-		g_key_file_set_string(config, "backupcopy", "time_fmt", text_time);
-		SETPTR(backupcopy_time_fmt, g_strdup(text_time));
-		if (enable_backupcopy)
+		g_key_file_set_integer(config, "untitled_document_save", "persistent_untitled_documents_interval_ms", persistent_docs_updater_interval_ms);
+		/* If radio button (not boolean variable, which is not updated yet at this moment) is active
+			- we check for target dir validity */
+		if (gtk_toggle_button_get_active(GTK_TOGGLE_BUTTON(pref_widgets.untitled_doc_persistent_radio)))
 		{
-			if (!EMPTY(backupcopy_text_dir) && store_target_directory(
-					backupcopy_text_dir, &backupcopy_backup_dir))
+			if (!EMPTY(persistent_docs_text_dir)
+					&& g_str_has_suffix(persistent_docs_text_dir, G_DIR_SEPARATOR_S))
 			{
-				g_key_file_set_string(config, "backupcopy", "backup_dir", backupcopy_text_dir);
+				/* If target dir path ends with dir separator - we consider it invalid */
```

Comment:
Why? If it's a problem somewhere, it can be corrected by the plugin easily:
```
persistent_docs_text_dir[strlen(persistent_docs_text_dir) - 1] = '\0';
```